proto/exocore/operator/v1/tx.proto

```diff
@@ -45,6 +46,14 @@ message OperatorInfo {
   string operator_meta_info = 3;
   // client_chain_earning_addr_list is the client chain earning address list.
   ClientChainEarningAddrList client_chain_earnings_addr = 4;
+  // commission defines the commission parameters.
+  cosmos.staking.v1beta1.Commission commission = 5 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+  // min_self_delegation is the validator's self declared minimum self delegation.
+  string min_self_delegation = 6 [
```

Comment:
Shouldn't MinSelfDelegation be a property of the AVS and not of the operator?

Reply:
In the cosmos-SDK, this is configured by the validator, the delegator can choose the validator according to this. So I add this field in the operator. It's property to configure a MinSelfDelegation in the AVS parameter to set a threshold for opt-in.
Do you think operators still need to have a MinSelfDelegation that must be greater than or equal to the MinSelfDelegation specified in the AVS parameters in order to attract delegators?

Reply:
There are two things here:
- Operator-level minimum self delegation, which is used by the operator to attract delegations. If the operator's self-delegation falls below this level, they are kicked out of the validator set. If we are to implement this, we should implement this in USD value as well. If, at the end of an epoch, the self-delegated value falls below the USD amount, the operator is not considered for inclusion in the validator set for any AVS.
- AVS-level minimum self-delegation, which is used by the AVS to ensure its operators have some skin in the game. This, too, must be denominated in USD (unfortunately). This is because we intend to support restaking on multiple chains, and validators from chain A (say Ethereum) will not necessarily be validators on chain B (say Avalanche). If we denominate the minimum self-stake in native restaked ETH and separately in native restaked AVAX, a validator active only on Ethereum but not on Avalanche will not be able to come up with any native restaked AVAX to meet the minimum. We can apply this mechanism similar to the operator-level implementation. If, at the end of an epoch, the self-delegated value falls below the USD amount required by that AVS, the operator is not considered for inclusion in the validator set for that AVS.

Reply:
As we discussed, we can only retain the `MinSelfDelegation` of AVS.